data/categories.txt

```diff
@@ -2440,7 +2440,7 @@ sw:^Makumbusho|kivutio|utalii
 historic-ruins|@tourism
 en:^Ruins|attraction|tourism|U+1F5FF
 ru:^Руины|развалины|достопримечательность|туризм
-ar:^آثار|أماكن الجذب|مناظر
+ar:^آثار|أماكن جذابة|مناظر|سياحة
```

Comment:
1. Yes, The "ruins" here is somewhere!
2.Yes the '^' points to the main translation .. and we can say it exist Immediately before the main translation, not left of the required word because the Arabic language typing from Right-to-Left.